cmd/snap/wait.go

```diff
@@ -115,17 +115,35 @@ func (wmx waitMixin) wait(id string) (*client.Change, error) {
 			tMax = time.Time{}
 		}
 
+		maybeShowLog := func(t *client.Task) {
+			nowLog := lastLogStr(t.Log)
+			if lastLog[t.ID] != nowLog {
+				pb.Notify(nowLog)
+				lastLog[t.ID] = nowLog
+			}
+		}
+
+		// Tasks in "wait" state communicate the wait reason
+		// via the log mechanism. So make sure the log is
+		// visible even if the normal progress reporting
+		// has tasks in "Doing" state (like "check-refresh")
+		// that would supress displaying the log. This will
```

Comment:
This failed the static checks

```suggestion
		// that would suppress displaying the log. This will
```